Homework_8_classes_objects_fields_methods/MyAmazingRPG/src/Main.java

```diff
@@ -0,0 +1,54 @@
+public class Main {
+    public static void main(String[] args) {
+
+
+        Character sirius = new Character();
+        Character titanarm = new Character();
+
+
+        sirius.name = "SIRIUS";
+        sirius.health = 87;
+        sirius.strength = 19;
+        sirius.defense = 7;
+        sirius.criticalHit = sirius.strength + 20;
```

Comment:
Looks like the critical hit can be just initialized with some value (without taking strength as a basis)

In your method you're already adding crit points to the strength points when hitting someone

```java
 public void useCriticalHit(Character other) {
        other.takeDamage(this.strength + this.criticalHit - other.defense);
}
```